ports/gts/portfile.cmake

```
@@ -1,29 +1,34 @@
-vcpkg_check_linkage(ONLY_DYNAMIC_LIBRARY ONLY_DYNAMIC_CRT)
+if(VCPKG_LIBRARY_LINKAGE STREQUAL "static")
+    set(patches fix-dllexport.patch)
+else()
+    set(patches "")
+endif()
 
-vcpkg_from_github(
-    OUT_SOURCE_PATH SOURCE_PATH
-    REPO  finetjul/gts
-    REF c4da61ae075f355d9ecc9f2d4767acf777f54c2b #0.7.6
-    SHA512 e53d11213c26cbda08ae62e6388aee0a14d2884de72268ad25d10a23e77baa53a2b1151c5cc7643b059ded82b8edf0da79144c3108949fdc515168cac13ffca9
-    HEAD_REF master
-    PATCHES
-        fix-M_PI-in-windows.patch
-        support-unix.patch
-        fix-pkgconfig.patch
-        glib2.patch
+vcpkg_from_sourceforge(
+    OUT_SOURCE_PATH src
```

```suggestion
    OUT_SOURCE_PATH SOURCE_PATH
```

This should be `SOURCE_PATH` by [convention](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/vcpkg_from_sourceforge.md#parameters).